app/models/container_group.rb

```diff
@@ -61,7 +61,7 @@ def event_where_clause(assoc = :ems_events)
     when :ems_events
       # TODO: improve relationship using the id
       ["container_namespace = ? AND container_group_name = ? AND #{events_table_name(assoc)}.ems_id = ?",
-       container_project.name, name, ems_id]
+       container_project.try(:name), name, ems_id]
```

Comment:
This isn't a must, it prevents the summary page of deleted group not to crash.
Technically there isnt a scenario in which a user can access this page but its easier for developing.